app/views/vms/edit_config.html.erb

```diff
@@ -1,56 +1,35 @@
 <h1>Configuring IP and DNS for VM '<%= @vm.name %>'</h1>
 
-<%= form_with(scope: :vm_info, model: @vm.config, url: {:controller => "vms", :action => "update_config", id: @vm.config.name}, local: true, builder: HartFormBuilder) do |form| %>
-  <% if @vm.request %>
-    <div class="field">
-      <div>
-        <strong>Operating System</strong> <br>
-        <%= label "Operating System", @vm.request.operating_system %>
-      </div>
-      <% unless @vm.request.comment.blank? %>
-        <div>
-          <strong>Comment</strong><br>
-          <%= label "Comment", @vm.request.comment %>
-        </div>
-      <% end %>
-    </div>
-  <% end %>
-
-  <div class="field">
-    <p><strong>Network Adapters</strong></p>
-    <table class="table">
-      <tr>
-        <th>Adapter</th>
-        <th>MAC Address</th>
-      </tr>
-      <% @vm.macs.each do |each| %>
-        <tr>
-          <%# Adapter name %>
-          <td><%= each.first %></td>
-          <td>
-            <%# MAC address %>
-            <span id="macAddress"><%= each.second %></span>
-            <%# Copy to clipboard button, JS matches on 'clipboard-btn' class %>
-            <span
-              class="btn btn-outline-primary btn-sm fa fa-copy clipboard-btn"
-              title="Copy MAC address to clipboard"
-              data-clipboard-text="<%= each.second %>">
-            </span>
-          </td>
-        </tr>
-      <% end %>
-    </table>
-  </div>
-
-  <div class="field">
-    <%= form.labeled_text_field 'IP', :ip %>
-  </div>
+<% if @vm.request %>
+  <h5>Operating System</h5>
+  <dl>
+    <dt>Name</dt>
+    <dd><%= @vm.request.operating_system %></dd>
+    <% unless @vm.request.comment.blank? %>
+      <dt>Comment</dt>
+      <dd><%= @vm.request.comment %></dd>
+    <% end %>
+  </dl>
+<% end %>
 
-  <div class="field">
-    <%= form.labeled_text_field 'DNS', :dns %>
-  </div>
+<h5>Network Adapters</h5>
+<dl>
+  <% @vm.macs.each do |adapter_name, mac_address| %>
+    <dt><%= adapter_name %></dt>
+    <dd>
+      <%= mac_address %>
+      <span
+        class="btn btn-outline-primary btn-sm fa fa-copy clipboard-btn"
```

Comment:
Readd comment `<%# Copy to clipboard button, JS matches on 'clipboard-btn' class %>`